src/index.html

```diff
@@ -7,7 +7,53 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
there shouldn't be such modificator:
![image](https://github.com/mate-academy/layout_stars/assets/86923089/91433a60-306a-4223-be5b-c37cd8e12b86)

you should handle it only using `stars` block modificator, e.g. `--1`